app/services/classification/matches_generator.rb

```diff
@@ -16,11 +16,12 @@ def matches
       possible_value_orders.each do |rule|
         match = generate_match(rule)
         matches << match.join(LookupKey::KEY_DELM)
-
-        if add_hostgroup_matches?(rule)
-          hostgroup_matches.each do |hostgroup_match|
-            match[match.index { |m| m =~ /hostgroup\s*=/ }] = hostgroup_match
-            matches << match.join(LookupKey::KEY_DELM)
+        ['hostgroup', 'organization', 'location'].each do |element|
```

Comment:
Instead of using hard-coded list in `matches_generator and values_hash_query`, would it be possible to declare constant which is having this set?